src/components/SearchModal/ImportToken.tsx

```diff
@@ -97,7 +99,9 @@ export function ImportToken({ tokens, list, onBack, onDismiss, handleCurrencySel
                 </AutoColumn>
                 {chainId && (
                   <ExternalLink href={getExplorerLink(chainId, token.address, ExplorerDataType.ADDRESS)}>
-                    <AddressText fontSize={12}>{token.address}</AddressText>
+                    <AddressText fontSize={12}>
+                      {IS_WITHIN_IFRAME ? shortenAddress(token.address) : token.address}
```

Comment:
can you add word-break styling to the `AddressText` just in case? could also happen on small devices